File: Telecom.Entities/Response/FaturaResponse.cs
namespace Telecom.Entities.Response
{
    public class FaturaResponse
    {

        public FaturaResponse()
        {
            
        }
        public FaturaResponse(int id, int contratoId, decimal valorCobrado, string status)
        {
            Id = id;
            ContratoId = contratoId;
            ValorCobrado = valorCobrado;
            Status = status;
        }

        public int Id { get; set; }                
        public int ContratoId { get; set; }
        public decimal ValorCobrado { get; set; }  
        public string? Status { get; set; }

    }

}

File: Telecom.Web/angular/telecom-front/src/app/entities/Response/FaturaResponse.ts
import { ContratoResponse } from "./ContratoResponse";
import { OperadoraServicoResponse } from "./OperadoraServicoResponse";

export interface FaturaResponse {
DataVencimento: string,
DataEmissao: string,
id: number,
contratoId: number,
valorCobrado: number,
status: string,
Contrato? : ContratoResponse,
Operadora?: OperadoraServicoResponse,

}
  